Backend/Backend/Controllers/ProfileController.cs

```diff
@@ -82,23 +83,28 @@ public async Task<ActionResult> SetupProfile([FromForm] ProfileSetupRequest setu
     {
         try
         {
-            this.LogDebugControllerAPICall(_logger, callerName: nameof(SetupProfile));
+            this.LogDebugControllerAPICall(_logger);
 
             // Identify User from JWT Token
             User? user = await _authService.IdentifyUserAsync(HttpContext);
 
             // If User is not found, return 404
             if (user == null)
                 return NotFound("User does not exist.");
-
+            
+            // Validate file extension
+            if (!ValidateAvatar(setupRequest.Avatar, out var errors)) {
+                return BadRequest(errors);
+            }
+            
             // Update User Profile
             bool isChanged = await _profileService.SetupProfileAsync(setupRequest, user);
 
             return isChanged ? Ok("User Profile Updated.") : Ok("User Profile Unchanged.");
         }
         catch (Exception e)
         {
-            this.LogErrorAPICall(_logger, e, callerName: nameof(SetupProfile));
+            this.LogErrorAPICall(_logger, e);
```

Comment:
Does this actually print the correct endpoint?

Reply:
Yes it does. Even in IDE it is "grayed out" because it is redundant